IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcTransportElementType/DocEntity.xml

```diff
@@ -6,22 +6,19 @@
 		<DocLocalization Locale="fr" Name="Type d&apos;équipement de transport" />
 	</Localization>
 	<Attributes>
-		<DocAttribute Name="PredefinedType" UniqueId="59d54e7c-05ef-4b69-8197-5d552e63c8f6" DefinedType="IfcTransportElementTypeSelect">
-			<Documentation>Identifies the predefined type of a transport element. This type may associate additional specific property sets.</Documentation>
+		<DocAttribute Name="PredefinedType" UniqueId="eb1df5a3-4b56-44e6-bd53-f7392d577eb3" DefinedType="IfcTransportElementTypeEnum">
```

Comment:
I think that the documentation could specify that this is a " a generalization of all **fixed** transport related objects".